services/hierarchy/src/main/kotlin/ProductService.kt

```diff
@@ -21,6 +21,7 @@ package org.eclipse.apoapsis.ortserver.services
 
 import org.eclipse.apoapsis.ortserver.dao.dbQuery
 import org.eclipse.apoapsis.ortserver.dao.dbQueryCatching
+import org.eclipse.apoapsis.ortserver.dao.repositories.repository.RepositoriesTable
```

Comment:
Commit message nit: "a product" -> "products" / "multiple products"
Is there actually a use case for providing multiple product ids? Otherwise the function could be simplified.

Reply:
Yeah I was gonna reuse this on the organization level, but now I realized that I should just get the repository ids in that case with a single query instead of getting the product ids first, and then using this. So I think I'll refactor this to only take one id.